templates/github/.github/workflows/publish.yml.j2

```diff
@@ -56,12 +56,6 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
-
-      {%- if deploy_client_to_rubygems %}
-      {{ run_script(name="Install Ruby client", file="install_ruby_client.sh", withenv=False) | indent(6) }}
-      {%- endif %}
-
```

Comment:
I think, you can/should remove the whole "build-bindings-docs" job from the workflow.

Reply:
I, hopefully, removed the whole section. I need your expertise on verifying that.